varats/varats/projects/test_projects/test_suite.py

```diff
@@ -0,0 +1,93 @@
+"""Project to run tests from the micro-benchmark Test-Suite."""
+import os
+import typing as tp
+
+import benchbuild as bb
+from plumbum import local
+
+from varats.project.project_util import (
+    ProjectBinaryWrapper,
+    wrap_paths_to_binaries,
+    BinaryType,
+)
+
+
+class MicroBenchmark(bb.Project):  # type: ignore
+    """
+    Get examples from the Test-Suite micro-benchmark suite.
+
+    Which can be found at https://github.com/SVF-tools/Test-Suite.
+    """
+
+    NAME = 'test_suite'
+    GROUP = 'test_projects'
+    DOMAIN = 'testing'
+
+    SOURCE = [
+        bb.source.Git(
+            remote="https://github.com/SVF-tools/Test-Suite.git",
+            local="svf-test-suite",
+            refspec="HEAD",
+            limit=1,  # None,
+            shallow=False,
+        )
+    ]
+
+    DIRS = [
+        "basic_c_tests",
+        "basic_cpp_tests",
+        # "complex_tests",
+        "cpp_types",
+        "cs_tests",
+        "fs_tests",
+        # "fstbhc_tests",
+        "path_tests",
+        "graphtxt",
+        # "mem_leak",
+        # "mta",
+        "non_annotated_tests",
+        "path_tests",
+    ]
+
+    FILE_PATHS = []  # type: tp.List[str]
```

Comment:
```suggestion
    FILE_PATHS: tp.List[Path] = []
```
easier to work with later on